### PR TITLE
[backstage] deploy to staging

### PR DESCRIPTION
**Pull request by greenboxal**

### CODE REVIEW

File: ci/lib/env.sh [hunk 1]
```diff
@@ -93,9 +93,9 @@ read -r -a BAZEL_CONFIG <<< "${BAZEL_CONFIG:-}"
 
 if is_ci; then
   BAZEL_CONFIG+=("ci")
-  RELEASE_ENV="${RELEASE_ENV:-dev}"
-  RELEASE_CLUSTER="${RELEASE_CLUSTER:-dev}"
-  RELEASE_NAMESPACE="${RELEASE_NAMESPACE:-pre-7a8311ad}"
```
Comment (vamsikc) [inline ci/lib/env.sh:98]: Curious, now that this is no longer hard-coded, where is this namespace fetched from for the dev deployment?

**Review by vamsikc — APPROVED** [review]
LGTM (Pending permissions fix)

File: ci/lib/env.sh (hunk 1 above)
Comment (greenboxal) [reply ci/lib/env.sh:98]: It's still hard coded to staging, do you mean on branch deployments?

Comment (vamsikc) [reply ci/lib/env.sh:98]: Yeah so for staging it makes sense as its hardcoded to staging.
But for the dev environment (dev.jsonnet), how does it know what PE namespace to deploy to?

Comment (greenboxal) [reply ci/lib/env.sh:98]: oh I see, it would be generated the same way as in credit_card by hashing the branch name, unless we want a different mechanism

Comment (greenboxal) [reply ci/lib/env.sh:98]: did you expect to still be able to deploy to a PE? I didn't add it given it won't match the PE from the other repos

Comment (vamsikc) [reply ci/lib/env.sh:98]: ah got it makes sense, I misunderstood - no strict need to deploy to PE, staging is definitely sufficient 👍

**Pull request merged**